### ISSUE TITLE
Não esta salvando 

### ISSUE THREAD

**Issue by kahbeltcher** [UNIVALI-LITE/Portugol-Studio#961]
**Descrição do Problema**:
eu crio um programa e ele não salva e as bibliotecas também demoram demais a carregar isso quando carregam.

**Mensagem de Erro (se houver)**:

**Captura de Tela / Arquivo.por**:

**Especificações do sistema**:

**Comment by AdsonEsteves** [comment]
@kahbeltcher 
Qual seu sistema operacional? Pode colocar o tamanho de memoria RAM?
O Portugol chega a apresentar alguma mensagem de erro?
Quando vc diz que as bibliotecas demoram a carregar, fala sobre a hora da execução de um código?

**Comment by kahbeltcher** [comment]
Não é na hora de rodar ,mas na hora de montar o código em si que a
biblioteca fica demorando demais a carregar.
4.GB de ram
Meu notebook é um idepad da lenovo de 1tb.
Não mostra nenhuma mensagem de erro não. O que aconteceu foi que ontem .
Eu estava montando um programa e eu salvei só que depois fui procurar o
arquivo que tinha salvo e acabou não salvando aí tive que montar de
novo,MAS eu tenho certeza de ter salvo.
outra coisa que aconteceu hoje.
dando erro mas nao informa qual

Em ter, 20 de jul de 2021 01:50, Adsu ***@***.***> escreveu:

**Comment by kahbeltcher** [comment]
> @kahbeltcher
> Qual seu sistema operacional? Pode colocar o tamanho de memoria RAM? 4GB
> O Portugol chega a apresentar alguma mensagem de erro? Hoje ele mostrou um erro sim no meu codigo. já enviei o print por e-mail
> Quando vc diz que as bibliotecas demoram a carregar, fala sobre a hora da execução de um código?
Não,na hora da montagem mesmo do código demora um bocado para carregar.

**Comment by kahbeltcher** [comment]
Não é na hora de rodar ,mas na hora de montar o código em si que a
biblioteca fica demorando demais a carregar.
4.GB <http://4.gb/> de ram
Meu notebook é um idepad da lenovo de 1tb.
Não mostra nenhuma mensagem de erro não. O que aconteceu foi que ontem .
Eu estava montando um programa e eu salvei só que depois fui procurar o
arquivo que tinha salvo e acabou não salvando aí tive que montar de
novo,MAS eu tenho certeza de ter salvo.


Em ter., 20 de jul. de 2021 às 00:50, Adsu ***@***.***>
escreveu:

**Comment by Markfgui675** [comment]
Bom dia, @kahbeltcher ! Você poderia tirar um print do erro? Acho que dessa forma ficaria mais fácil para eu te ajudar.
